README.md

```diff
@@ -1,7 +1,7 @@
 # Envoy filter example
 
 This project demonstrates the linking of additional filters with the Envoy binary.
-A new filter `echo2` is introduced, identical modulo renaming to the existing
+A new filter `my.net.filter.reader.example` is introduced, identical modulo renaming to the existing
```

Comment:
Can you use reverse DNS, e.g. `net.some.filter.reader.example? This is the preferred convention.

Reply:
Got it. So basically I have my own script (written in js using yeoman, not sure if the community like it haha. Apparently I need to work on my python skill to have it in `./tools` dir) something like:

```
$ ./generate net.reader --package net.some.filter.reader.example
$ ./generate http.encoder --package http.some.filter.encoder.example
```

to scaffold things out.